Doc: trip delete (#3081)
CanalTP/navitia: documentation/slate/source/includes/objects.md

|id|string|The id of the trip|
|name|string|The name of the trip|

It encapsulates many instances of vehicle_journey.

### Vehicle-journey

A vehicle-journey describes a scheduled vehicle circulation, the days on which it circulates according to base-schedule,
the days it circulates according to realtime information.

> Note that multiple vehicle-journeys are often associated to the same trip for technical and logical

Reviewer: ```suggestion
Note that multiple vehicle-journeys are often associated with the same trip for technical and logical
```

> reasons (for the changes of daylight saving time, for the changes after applying realtime disruptions, etc.).

Reviewer: ```suggestion
reasons (for changes regarding daylight saving time or after applying realtime disruptions, etc.).
```
